com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardNodeDefinitions.cs

```diff
@@ -32,7 +32,14 @@ internal class StandardNodeDefinitions
                     Usage.Out
                 )
             }
-        )
+        ),
+
+        new FunctionDescriptor(
+            1,
+            "Example",
+            "Out = In;",
+            new ParameterDescriptor("In", TYPE.Any, Usage.In, 1, 1, 0, 1),
```

Comment:
If you wrote in a 4x4 identity matrix, it would only be an identity for mat4. Scalar/Vector values still make sense, but if we get a matrix of any other size the string of values would no longer be an identity unless we treated it as two-dimensional data and copied the identity submatrix of the 4x4 for other widths/heights.